oci.go

```diff
@@ -302,3 +310,38 @@ func noNeedForOutput(detach bool, tty bool) bool {
 
 	return true
 }
+
+func getCgroupsDirPath() (string, error) {
+	if cgroupsDirPath != "" {
+		return cgroupsDirPath, nil
+	}
+
+	f, err := os.Open("/proc/self/mountinfo")
```

Comment:
Can you create a global `var` for this file and add a test for this function? By using a `var` you can write a test that changes the value of it to a temp file. See for example what we currently do for `/proc/cpuinfo`:

- https://github.com/clearcontainers/runtime/blob/master/cc-check.go#L88..L103

And here's how we test it (we check what happens when the file doesn't exist, when it contains invalid data and when it contains valid data):

- https://github.com/clearcontainers/runtime/blob/master/cc-check_test.go#L47..L94

Reply:
sure, I will read it